### ISSUE TITLE
Implement POST /projects/:projectID/files, /files

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/publish.webmaker.org#36]
Create a file, by sending the following message body:
```javascript
{
   name: ...,
   path: ...
}
```

**Comment by ashleygwilliams** [comment]
there will be 2 routes to create new files:

```
POST projects/:project_id/files
POST files/
````

should receive:
```
{
   path: String,
   project_id: Integer,
   buffer: Blob
}
```

**Comment by ashleygwilliams** [comment]
this is solved by #44
